client-ci-analysis/ci-failure-analysis/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ci_failure_analysis
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var jsonFiles = new[]
+            {
+                @"private_builds.json",
+                @"official_builds.json"
```

Comment:
Yes, the files are the ones created by `find-builds-in-sprint`. Note that the entropy repo is just a dumping ground, not production quality code. Making it write files, rather than write to output is an obvious improvement, but so far I've just been copy/pasting. Maybe I'll make the improvement next sprint when I need to use the tool again.